event-handler/cli.go

```diff
@@ -123,6 +123,12 @@ type IngestConfig struct {
 	// Types are event types to log
 	Types []string `help:"Comma-separated list of event types to forward" env:"FDFWD_TYPES"`
 
+	// SkipTypesRaw are event types to skip
+	SkipTypesRaw []string `name:"skip-types" help:"Comma-separated list of event types to skip" env:"FDFWD_SKIP_TYPES"`
+
+	// SkipSessionTypes is a map generated from SkipTypesRaw
+	SkipTypes map[string]struct{} `kong:"-"`
```

Comment:
Nit: For consistency with `SkipSessionTypes`.
```suggestion
	// SkipEventTypesRaw are event types to skip
	SkipEventTypesRaw []string `name:"skip-event-types" help:"Comma-separated list of event types to skip" env:"FDFWD_SKIP_EVENT_TYPES"`

	// SkipEventTypes is a map generated from SkipEventTypesRaw
	SkipEventTypes map[string]struct{} `kong:"-"`
```